docs/detections/rules-ui-create.asciidoc

```diff
@@ -266,9 +266,13 @@ NOTE: To preview rules, you need the `read` privilege to the `.preview.alerts-se
 To preview a rule:
 
 . Write the rule query.
+. Choose how you want to preview the query results:
+
+** *Quick query preview*: Allows you to select from pre-defined timeframes -- *Last hour*, *Last day*, or *Last month* -- when previewing rule results. Note that threshold and event correlation rules have limited timeframe options. The rule interval and look-back time are also pre-defined for the preview and differ by rule type. These settings cannot be modified.
+** *Advanced query preview*: Allows you to choose a custom timeframe for the rule preview, schedule how often the rule should run, and specify a look-back time.
 +
-. Select a timeframe of data to preview query results -- *Last hour*, *Last day*, or *Last month* -- from the *Quick query preview* drop-down.
-+
+TIP: It is recommended to set the `Additional look-back time` to at least 1 minute. This ensures there are no missing alerts when a rule does not run exactly at its scheduled time. In addition, avoid setting long timeframes with short rule intervals. This might cause the rule preview to timeout.
```

Comment:
```suggestion
TIP: Avoid setting long timeframes with short rule intervals. This might cause the rule preview to timeout.
```
In preview, alerts won't be missed due to scheduling drift so the additional lookback time is not strictly necessary. It's there for users to select so they can match the real desired parameters as closely as possible, and it still has an effect on how the rule executes, but we can leave out the recommendation here IMO.